data/src/main/java/io/gnosis/data/repositories/TransactionRepository.kt

```diff
@@ -15,11 +18,15 @@ class TransactionRepository(
     private val gatewayApi: GatewayApi
 ) {
 
-    suspend fun getTransactions(safeAddress: Solidity.Address): Page<Transaction> =
-        gatewayApi.loadTransactions(safeAddress.asEthereumAddressChecksumString())
+    suspend fun getTransactions(safeAddress: Solidity.Address): Page<UnifiedEntry> {
```

Comment:
We should have separate calls for getting queued and history transactions.

Reply:
Sure. But should that be part of this PR?

Reply:
Done in 1a1106d143e60c2c652dd3b48988ec1bcba1e75b